payload.go

```diff
@@ -128,7 +128,13 @@ func (p *payload) makeSession() *sessionJSON {
 
 func (p *payload) severityReasonPayload() *severityReasonJSON {
 	if reason := p.handledState.SeverityReason; reason != "" {
-		return &severityReasonJSON{Type: reason}
+		json := &severityReasonJSON{
+			Type: reason, Attributes: make(map[string]string, 1),
```

Comment:
Is there any benefit to not `make`-ing this map unless we need it?

Reply:
That's a good question - its one allocation we could skip. I'll move it around. 👍